packages/connect-query-core/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@connectrpc/connect-query-core",
```

Comment:
Looks like the turbo cache and several other things end up in the package that should not be there:

```
$ pnpm pack
$ tar -tf connectrpc-connect-query-core-2.0.0-rc.2.tgz 
package/.eslintignore
package/dist/cjs/gen/bigint_pb.js
package/dist/esm/gen/bigint_pb.js
package/dist/cjs/call-unary-method.js
package/dist/esm/call-unary-method.js
package/dist/cjs/connect-query-key.js
package/dist/esm/connect-query-key.js
package/dist/cjs/create-infinite-query-options.js
package/dist/esm/create-infinite-query-options.js
package/dist/cjs/create-query-options.js
package/dist/esm/create-query-options.js
package/dist/cjs/gen/eliza_pb.js
package/dist/esm/gen/eliza_pb.js
package/dist/cjs/index.js
package/dist/esm/index.js
package/dist/cjs/gen/list_pb.js
package/dist/esm/gen/list_pb.js
package/dist/cjs/message-key.js
package/dist/esm/message-key.js
package/dist/cjs/gen/proto2_pb.js
package/dist/esm/gen/proto2_pb.js
package/dist/cjs/gen/proto3_pb.js
package/dist/esm/gen/proto3_pb.js
package/dist/cjs/structural-sharing.js
package/dist/esm/structural-sharing.js
package/dist/cjs/transport-key.js
package/dist/esm/transport-key.js
package/dist/cjs/utils.js
package/dist/esm/utils.js
package/dist/cjs/package.json
package/tsconfig.build.json
package/tsconfig.json
package/.turbo/turbo-attw.log
package/.turbo/turbo-build.log
package/.turbo/turbo-clean.log
package/.turbo/turbo-format.log
package/.turbo/turbo-generate.log
package/dist/cjs/gen/bigint_pb.d.ts.map
package/dist/esm/gen/bigint_pb.d.ts.map
package/dist/cjs/gen/bigint_pb.js.map
package/dist/esm/gen/bigint_pb.js.map
package/dist/cjs/call-unary-method.d.ts.map
package/dist/esm/call-unary-method.d.ts.map
package/dist/cjs/call-unary-method.js.map
package/dist/esm/call-unary-method.js.map
package/dist/cjs/connect-query-key.d.ts.map
package/dist/esm/connect-query-key.d.ts.map
package/dist/cjs/connect-query-key.js.map
package/dist/esm/connect-query-key.js.map
package/dist/cjs/create-infinite-query-options.d.ts.map
package/dist/esm/create-infinite-query-options.d.ts.map
package/dist/cjs/create-infinite-query-options.js.map
package/dist/esm/create-infinite-query-options.js.map
package/dist/cjs/create-query-options.d.ts.map
package/dist/esm/create-query-options.d.ts.map
package/dist/cjs/create-query-options.js.map
package/dist/esm/create-query-options.js.map
package/dist/cjs/gen/eliza_pb.d.ts.map
package/dist/esm/gen/eliza_pb.d.ts.map
package/dist/cjs/gen/eliza_pb.js.map
package/dist/esm/gen/eliza_pb.js.map
package/dist/cjs/index.d.ts.map
package/dist/esm/index.d.ts.map
package/dist/cjs/index.js.map
package/dist/esm/index.js.map
package/dist/cjs/gen/list_pb.d.ts.map
package/dist/esm/gen/list_pb.d.ts.map
package/dist/cjs/gen/list_pb.js.map
package/dist/esm/gen/list_pb.js.map
package/dist/cjs/message-key.d.ts.map
package/dist/esm/message-key.d.ts.map
package/dist/cjs/message-key.js.map
package/dist/esm/message-key.js.map
package/dist/cjs/gen/proto2_pb.d.ts.map
package/dist/esm/gen/proto2_pb.d.ts.map
package/dist/cjs/gen/proto2_pb.js.map
package/dist/esm/gen/proto2_pb.js.map
package/dist/cjs/gen/proto3_pb.d.ts.map
package/dist/esm/gen/proto3_pb.d.ts.map
package/dist/cjs/gen/proto3_pb.js.map
package/dist/esm/gen/proto3_pb.js.map
package/dist/cjs/structural-sharing.d.ts.map
package/dist/esm/structural-sharing.d.ts.map
package/dist/cjs/structural-sharing.js.map
package/dist/esm/structural-sharing.js.map
package/dist/cjs/transport-key.d.ts.map
package/dist/esm/transport-key.d.ts.map
package/dist/cjs/transport-key.js.map
package/dist/esm/transport-key.js.map
package/dist/cjs/utils.d.ts.map
package/dist/esm/utils.d.ts.map
package/dist/cjs/utils.js.map
package/dist/esm/utils.js.map
package/README.md
package/proto/bigint.proto
package/proto/eliza.proto
package/proto/list.proto
package/proto/proto2.proto
package/proto/proto3.proto
package/dist/cjs/gen/bigint_pb.d.ts
package/dist/esm/gen/bigint_pb.d.ts
package/src/gen/bigint_pb.ts
package/dist/cjs/call-unary-method.d.ts
package/dist/esm/call-unary-method.d.ts
package/src/call-unary-method.ts
package/dist/cjs/connect-query-key.d.ts
package/dist/esm/connect-query-key.d.ts
package/src/connect-query-key.test.ts
package/src/connect-query-key.ts
package/dist/cjs/create-infinite-query-options.d.ts
package/dist/esm/create-infinite-query-options.d.ts
package/src/create-infinite-query-options.ts
package/dist/cjs/create-query-options.d.ts
package/dist/esm/create-query-options.d.ts
package/src/create-query-options.test.ts
package/src/create-query-options.ts
package/dist/cjs/gen/eliza_pb.d.ts
package/dist/esm/gen/eliza_pb.d.ts
package/src/gen/eliza_pb.ts
package/dist/cjs/index.d.ts
package/dist/esm/index.d.ts
package/src/index.ts
package/dist/cjs/gen/list_pb.d.ts
package/dist/esm/gen/list_pb.d.ts
package/src/gen/list_pb.ts
package/dist/cjs/message-key.d.ts
package/dist/esm/message-key.d.ts
package/src/message-key.test.ts
package/src/message-key.ts
package/dist/cjs/gen/proto2_pb.d.ts
package/dist/esm/gen/proto2_pb.d.ts
package/src/gen/proto2_pb.ts
package/dist/cjs/gen/proto3_pb.d.ts
package/dist/esm/gen/proto3_pb.d.ts
package/src/gen/proto3_pb.ts
package/dist/cjs/structural-sharing.d.ts
package/dist/esm/structural-sharing.d.ts
package/src/structural-sharing.test.ts
package/src/structural-sharing.ts
package/dist/cjs/transport-key.d.ts
package/dist/esm/transport-key.d.ts
package/src/transport-key.test.ts
package/src/transport-key.ts
package/dist/cjs/utils.d.ts
package/dist/esm/utils.d.ts
package/src/utils.test.ts
package/src/utils.ts
package/vite.config.ts
package/src/test/test-utils.tsx
package/buf.gen.yaml
package/LICENSE
package/package.json
```

Reply:
woof, yeah I'll fix that.